app/presenters/finder_content_item_presenter.rb

```diff
@@ -50,6 +50,10 @@ def details
       summary: metadata.fetch("summary", nil),
       facets: schema.fetch("facets", []),
     }
+    unless metadata["default_order"].nil?
+      details_hash[:default_order] = metadata["default_order"]
```

Comment:
If anything'd be preferable, for me it'd be a `compact`-style `reject` to only have details with non-`nil` values returned.

I don't know https://github.com/alphagov/finder-frontend well enough to know if that'd be a good move, though.